------------------------------------------------------------
revno: 332
committer: Scott Moser <smoser@ubuntu.com>
branch nick: trunk
timestamp: Thu 2011-01-27 09:30:37 -0500
message:
  fix bug in fixing permission on default log file
--- a/cloudinit/__init__.py
+++ b/cloudinit/__init__.py
@@ -487,8 +487,8 @@
         log_file = cfg['def_log_file']
         fp = open(log_file,"ab")
         fp.close()
-    if log_file and 'syslog' in cfg:
-        perms = cfg['syslog']
+    if log_file and 'syslog_fix_perms' in cfg:
+        perms = cfg['syslog_fix_perms']
         (u,g) = perms.split(':',1)
         if u == "-1" or u == "None": u = None
         if g == "-1" or g == "None": g = None
